CHANGES.rst

```diff
@@ -22,6 +22,8 @@ New Features
 
 - ``astropy.cosmology``
 
+  - The default cosmology has been changed to Planck 2015
```

Comment:
Always the problem -- you have to make a PR to get a PR number to put in your PR.